service/src/main/kotlin/app/cash/backfila/service/BackfilaMetrics.kt

```diff
@@ -0,0 +1,90 @@
+package app.cash.backfila.service
+
+import javax.inject.Inject
+import javax.inject.Singleton
+import misk.metrics.Metrics
+
+@Singleton
+class BackfilaMetrics @Inject internal constructor(metrics: Metrics) {
+  val runBatchDuration = metrics.histogram(
+    name = "run_batch_duration",
```

Comment:
are these names namespaced in any way? Should I prefix them with all `backfila.`?

Reply:
I think the'd all get grouped by the service name somewhere anyway?

Reply:
@swankjesse any opinion on metric names?